### PR TITLE
Lots of Cameron and a little Jones

### PR DESCRIPTION
**Pull request by DrMerlin**
So... this looks drastic- but we've been using the 0.3.0 cookbook for the past 11 months.  So I'm going to assume we don't 'need' anything here.  It might be nice to merge the .kitchen.yml stuff though.

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -5,21 +5,9 @@ driver_config:
 
 platforms:
 - name: ubuntu-12.04
-  driver_config:
```
Comment (purgatorio) [inline .kitchen.yml]: Lets keep these in instead of removing them.

File: Berksfile [hunk 2]
```diff
@@ -2,6 +2,12 @@ site :opscode
```
Comment (purgatorio) [inline Berksfile]: This needs to be updated to Berkshelf 3

File: Gemfile [hunk 3]
```diff
@@ -4,5 +4,5 @@ gem 'rake'
 gem 'foodcritic'
 gem 'rubocop'
 gem 'berkshelf'
-gem 'test-kitchen', '~> 1.0.0.beta'
+gem 'test-kitchen', '~> 1.0'
```
Comment (purgatorio) [inline Gemfile]: Should be test-kitchen 1.2

File: test/cookbooks/mongodb-test/attributes/default.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+default['mongodb_test']['version'] = '2.4.5'
```
Comment (purgatorio) [inline test/cookbooks/mongodb-test/attributes/default.rb]: This is the wrong mongo version

File: test/cookbooks/mongodb-test/templates/default/sv-mongodb-run.erb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# This service template originates from the test cookbook
+exec 2>&1
```
Comment (purgatorio) [inline test/cookbooks/mongodb-test/templates/default/sv-mongodb-run.erb]: This is missing the ulimit changes you made.

File: test/integration/mongod/bats/mongod.bats [hunk 6]
```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env bats
```
Comment (purgatorio) [inline test/integration/mongod/bats/mongod.bats]: Why are we removing the bats test?

File: test/cookbooks/mongodb-test/templates/default/sv-mongodb-run.erb (hunk 5 above)
Comment (DrMerlin) [inline test/cookbooks/mongodb-test/templates/default/sv-mongodb-run.erb]: Ah- that’s in the test cookbook.  Weird.  I made changes to the template itself.  I will follow up with the changes in test.

File: test/integration/mongod/bats/mongod.bats (hunk 6 above)
Comment (DrMerlin) [inline test/integration/mongod/bats/mongod.bats]: Cameron rewrote it as the server spec.

Comment (purgatorio) [inline test/integration/mongod/bats/mongod.bats]: We can have both in there. Having the serverspec won't hurt anything, but we use the bats tests for our testing.

Comment (purgatorio) [inline test/integration/mongod/bats/mongod.bats]: Okay, we can have it use serverspec instead

**Pull request merged**